### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by CalebeNavarro**
#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [x] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/element/remove/index.md [hunk 1]
```diff
@@ -30,8 +30,8 @@ remove()
 ```
 
 ```js
-var el = document.getElementById('div-02');
-el.remove(); // Removes the div with the 'div-02' id
+let element = document.getElementById('div-02');
```
Comment (wbamberg) [inline files/en-us/web/api/element/remove/index.md:33]:
```suggestion
const element = document.getElementById('div-02');
```

(please see https://github.com/mdn/content/discussions/10389 )

**Review by wbamberg — CHANGES_REQUESTED** [review]
Thanks for your PR, @CalebeNavarro ! I think that `let` could be a `const`.

**Review by wbamberg — APPROVED** [review]
👍 thank you!

File: files/en-us/web/api/element/remove/index.md (hunk 1 above)
Comment (CalebeNavarro) [reply files/en-us/web/api/element/remove/index.md]: I read #10389 and make full sense, thank you!

**Pull request merged**